docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3"
```

Comment:
Why using a docker-compose and not directly a kubernetes manifest?

Reply:
even for testing I mean

Reply:
Because I like the simplicity of docker-compose for getting things up and down locally

Reply:
Ok